symfony/sendgrid-mailer/4.3/manifest.json

```diff
@@ -1,5 +1,5 @@
 {
     "env": {
-        "#1": "MAILER_DSN=smtp://$SENDGRID_KEY@sendgrid"
+        "#1": "MAILER_DSN=smtp://apikey:{YOUR-SENDGRID-API-KEY}@smtp.sendgrid.net"
```

Comment:
Actually, I think I'm misunderstanding this too - the `apikey` username looks like it should be provided by the transport class: https://github.com/symfony/sendgrid-mailer/blob/master/Smtp/SendgridTransport.php#L29